BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/MainActivity.java

```diff
@@ -42,13 +46,20 @@ protected void onCreate(Bundle savedInstanceState) {
                     Toast.makeText(getApplicationContext(),
                             "Created session!",
                             Toast.LENGTH_LONG).show();
-
+                    /*
```

Comment:
I still have to do the merge and this part ha been completely refactored so it will be removed at that point